ethclient/ethclient.go

```diff
@@ -121,6 +121,16 @@ func (ec *Client) getBlock(ctx context.Context, method string, args ...interface
 	if err := json.Unmarshal(raw, &body); err != nil {
 		return nil, err
 	}
+
+	var blockExtraData *[]byte
+	if len(body.BlockExtraData) != 0 {
+		blockExtraDataDecoded, err := hexutil.Decode(body.BlockExtraData)
+		if err != nil {
+			return nil, err
```

Comment:
Could you change this to `fmt.Errorf("problem parsing block extra data: %w", err)` so the error is clearer if there's ever a problem?